config/v1/types_apiserver.go

```diff
@@ -30,6 +30,15 @@ type APIServerSpec struct {
 	// - ConfigMap.Data["ca-bundle.crt"] - CA bundle.
 	// +optional
 	ClientCA ConfigMapNameReference `json:"clientCA"`
+	// extraCORSAllowedOrigins is a field of allowed CORS that should be added on top of the basic CORS needed by the
+	// infrastructure in order to allow in-cluster components to access the API and OAuth servers using Javascript.
```

Comment:
This is not specifically related to in-cluster components.

Reply:
Not sure if I would call out the OAuth server here.

Reply:
> Not sure if I would call out the OAuth server here.

Yes, he should indicate it because it's likely to come up as a question.

Reply:
> > Not sure if I would call out the OAuth server here.
> 
> Yes, he should indicate it because it's likely to come up as a question.

Okay.  "OAuth servers" sounds weird to me.  I could see "integrated OAuth server" or "integrated OAuth servers" even depending on how we want to think about future KeyCloak work.

Reply:
API and OAuth servers were two together, but the 's' in servers does sound weird, I'm going with `integrated OAuth server`.
Once the Keycloak operator is being written, this field should also be respected, but I don't think we need to specifically mention it here.